themes/_default.yaml

```diff
@@ -145,6 +145,21 @@ button:
     font-family: "var(--tx-generic-font-primary-font-family), var(--tx-generic-font-primary-generic-family)"
     hover-color: *primary
     hover-border-color: *primary
+  link:
+    color: *text
+    font:
+      family: "var(--tx-generic-font-secondary-font-family)"
+      size: "14px"
+      weight: "var(--tx-generic-font-primary-weight-regular)"
```

Comment:
By directly you mean the actual values? The way it is adds more flexibility per theme.